cristin-import/src/test/java/no/unit/nva/cristin/lambda/CristinEntryEventConsumerTest.java

```diff
@@ -287,14 +287,26 @@ public void handlerCreatesFileWithCustomNameWhenCristinIdIsNotFound() throws Jso
         assertThat(actualReport.getInput().getDetail().getContents(), is(equalTo(cristinObjectWithoutId)));
     }
 
-    @Disabled
     @Test
-    public void savesFileInInputFolderErrorExceptionNameInputFileLocationInputFileWhenFailingToSaveInDynamo() {
-        InputStream inputData = IoUtils.inputStreamFromResources("valid_cristin_event_throws_exception.json");
-        Executable action = () -> handler.handleRequest(inputData, outputStream, CONTEXT);
-        assertThrows(RuntimeException.class, action);
+    public void savesFileInInputFolderErrorExceptionNameInputFileLocationInputFileWhenFailingToSaveInDynamo()
+        throws Throwable {
+        CristinObject cristinObject = cristinDataGenerator.randomObject();
+        JsonNode cristinObjectWithCustomSecondaryCategory =
+            cristinDataGenerator.injectCustomSecondaryCategoryIntoCristinObject(
+                cristinObject, randomString());
+        AwsEventBridgeEvent<FileContentsEvent<JsonNode>> awsEvent =
+            cristinDataGenerator.toAwsEvent(cristinObjectWithCustomSecondaryCategory);
+        InputStream inputStream = IoUtils.stringToStream(awsEvent.toJsonString());
+        Executable action = () -> handler.handleRequest(inputStream, outputStream, CONTEXT);
+
+        Exception exception = assertThrows(RuntimeException.class, action);
+
         S3Driver s3Driver = new S3Driver(s3Client, "bucket");
-        String expectedErrorFileLocation = "errors/parent/child/filename.json/5709.json";
+        String expectedFilePath = awsEvent.getDetail().getFileUri().getPath();
+        String exceptionName = exception.getCause().getClass().getSimpleName();
+        String fileIdWithEnding = cristinObject.getId().toString() + ".json";
```

Comment:
use no.unit.nva.cristin.lambda.CristinEntryEventConsumer.FILE_ENDING;

Reply:
I have made the changes, but I am wondering wheter FILE_ENDING is a good name. Since its a constant with the value ".json" should it not specify that in its name to increase readability?

Reply:
rename it to JSON